ft_statistics_mvpa.m

```diff
@@ -0,0 +1,231 @@
+% regularisation parameters and the kernel. Hyperparameters can be set
+% using the cfg.param substruct. For instance, in LDA, cfg.param.lambda =
+% 'auto' sets the lambda regularisation parameter.
+%
+% The specification of the hyperparameters is found in the training function
+% for each classifier at github.com/treder/MVPA-Light/tree/master/classifier
+% If a hyperparameter is not specified, default values are used.
+%
+% SEARCHLIGHT ANALYSIS:
+% Classification using a feature searchlight approach can highlight which
+% feature(s) are informative. To this end, classification is performed on 
+% each feature separately. However, neighbouring features can enter the 
+% classification together when a matrix of size [features x features]
+% specifying the neighbours is provided. Additional parameters:
+% .nb          - [features x features] matrix specifying which features
+%                are neighbours of each other.
+%                          - EITHER - 
+%                a GRAPH consisting of 0's and 1's. A 1 in the 
+%                (i,j)-th element signifies that feature i and feature j 
+%                are neighbours, and a 0 means they are not neighbours
+%                            - OR -
+%                a DISTANCE MATRIX, where larger values mean larger distance.
+%                If no matrix is provided, every feature is only neighbour
+%                to itself and classification is performed for each feature 
+%                separately.
+% .size        - if a nb matrix is provided, size defines the 
+%                size of the 'neighbourhood' of a feature.
+%                if nb is a graph, it gives the number of steps taken 
+%                     through the nb matrix to find neighbours:
+%                     0: only the feature itself is considered (no neighbours)
+%                     1: the feature and its immediate neighbours
+%                     2: the feature, its neighbours, and its neighbours'
+%                     neighbours
+%                     3+: neighbours of neighbours of neighbours etc
+%                     (default 1)
+%                if nb is a distance matrix, size defines the number of
+%                     neighbouring features that enter the classification
+%                     0: only the feature itself is considered (no neighbours)
+%                     1: the feature and its first closest neighbour 
+%                        according to the distance matrix
+%                     2+: the 2 closest neighbours etc.
+%
+% -- TODO: for time x time generalisation, in MVPA light we can use two
+% different datasets (one for training the classifier, the other one for
+% testing). This could be realised eg using extra fields in cfg such as
+% cfg.X2 for dataset 2 and cfg.design2 for the second design matrix
+%
+% Returns:
+%   stat        = struct with results. the .metric field contains the
+%                   requested metrics
+% 
+% (c) matthias treder
```

Comment:
```suggestion

% Copyright (C) 2019, Matthias Treder
%
% This file is part of FieldTrip, see http://www.fieldtriptoolbox.org
% for the documentation and details.
%
%    FieldTrip is free software: you can redistribute it and/or modify
%    it under the terms of the GNU General Public License as published by
%    the Free Software Foundation, either version 3 of the License, or
%    (at your option) any later version.
%
%    FieldTrip is distributed in the hope that it will be useful,
%    but WITHOUT ANY WARRANTY; without even the implied warranty of
%    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
%    GNU General Public License for more details.
%
%    You should have received a copy of the GNU General Public License
%    along with FieldTrip. If not, see <http://www.gnu.org/licenses/>.
%
% $Id$

```